ext/serializer.c

```diff
@@ -1268,6 +1294,23 @@ void ddtrace_serialize_span_to_array(ddtrace_span_data *span, zval *array) {
         _add_assoc_zval_copy(el, "type", &prop_type_as_string);
     }
 
+    if (span_type) {
+        zend_hash_str_del(meta, ZEND_STRL("span.type"));
+    }
+
+    zval *analytics_event = zend_hash_str_find(meta, ZEND_STRL("analytics.event"));
+    if (analytics_event) {
+        zval analytics_event_as_double;
+        if (Z_TYPE_P(analytics_event) == IS_STRING) {
+            ZVAL_DOUBLE(&analytics_event_as_double, zend_is_true(analytics_event)); // 'true' => 1.0, false => 0.0
```

Comment:
Is `zend_is_true` what you want? The comment sounds like you want the string `"false"` to mean 0?

Reply:
Well, I personally don't want to; the system tests want to (or at least, they want `"true"` to mean 1)

Reply:
More specifically, they want it to mean `"1"`, but I personally don't agree with that as this is inconsistent with the other metrics